src/docfx/build/Build.cs

```diff
@@ -111,6 +114,12 @@ bool ShouldBuildFile(Document file)
                         return false;
                     }
 
+                    if (file.ContentType == ContentType.TableOfContents)
+                    {
+                        tocFiles.Add(file);
+                        return false;
+                    }
```

Comment:
I don't think that's a good way to do that, make the input and output really not clear